### PR TITLE
Fixes #656 Translation dropdown

### LINKED ISSUE #656: Update translation dropdown, v3 has added couple more plus ids are changed!
Available translations: http://staging.quran.com:3000/api/v3/options/translations

### PR DESCRIPTION
**Pull request by mmahalwy**
Also:
Fixes https://github.com/quran/quran.com-frontend/issues/681

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32815

File: src/server.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import createHistory from 'react-router/lib/createMemoryHistory';
 import { Provider } from 'react-redux';
 import { IntlProvider } from 'react-intl';
 import cookie from 'react-cookie';
-import raven from 'raven';
+import Raven from 'raven';
```
Comment (mmahalwy) [inline src/server.js]: sorry, this is probably unrelated to the PR. Happy to create a new one. But it's scoped to this file.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32816

**Comment by naveed-ahmad** [comment]
@mmahalwy we've multiple translation for many languages now. How about adding author name beside language name ?

**Comment by mmahalwy** [comment]
@naveed-ahmad +1. I can add it

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32771

**Comment by mmahalwy** [comment]
Going to merge this to fix things, and create issue from @naveed-ahmad comment

**Pull request merged**